utils/nctl/sh/scenarios-upgrades/upgrade_scenario_09.sh

```diff
@@ -163,46 +169,33 @@ function _step_05()
     done
 }
 
-# Step 06: Await 1 era.
-function _step_06()
```

Comment:
Was there a reason to change the format that the test is using?

Reply:
Initially I saw no point in having them as separate functions, but it makes sense for convention. Added the step functions back in https://github.com/casper-network/casper-node/pull/3644/commits/e8c82e43f80f8f865ed042d1b0d044761a9651be